app/code/Magento/Bundle/Helper/Data.php

```diff
@@ -38,6 +38,6 @@ public function getAllowedSelectionTypes()
     {
         $configData = $this->config->getType(\Magento\Catalog\Model\Product\Type::TYPE_BUNDLE);
 
-        return isset($configData['allowed_selection_types']) ? $configData['allowed_selection_types'] : [];
+        return $configData['allowed_selection_types'] ?? : [];
```

Comment:
that is not valid syntax, and actualy a lot of test fail with this changes
![DeepinScreenshot_select-area_20200210153555](https://user-images.githubusercontent.com/2481206/74154181-ddaa2f00-4c1a-11ea-8867-a59be1c673fe.png)

Reply:
I fixed.